apps/bluetooth-music/README.md

```diff
@@ -0,0 +1,49 @@
+# Bluetooth music player
+
+    This is the app for play bluetooth music.
+
+## App URL
+
+* scheme and host:
+
+  * `yoda-app://bluetooth-music`
+
+* paths and queries:
+
+  * `start`: Start/resume play.
+
+  * `pause`: Pause music.
+
+  * `stop`: Stop music.
+
+  * `next`: Play next song.
+
+  * `prev`: Play previous song.
+
+  * `like`: Add current song to favorite.
+
+  * `info`: Query current song's information.
+
+|scheme and host|path|query's key|query's value|remark|
+|---|---|---|---|---|
+||start|||Start/resume play.|
+||pause|||Pause music.|
+||stop|||Stop music.|
+|yoda-app://bluetooth-music|next|||Play next song.|
+||prev|||Play previous song.|
+||like|||Add current song to favorite.|
+||info|||Query current song's information.|
```

Comment:
What's the table for?

Reply:
Use table to show more clearly URI contents